contents/blog/posthog-vs-matomo.md

```diff
@@ -0,0 +1,435 @@
+
+Matomo is primarily pitched as a Google Analytics alternative for marketing analytics, and offers useful tools for migrating GA data into Matomo for a seamless transition. Its focus on session tracking makes it ideal for running analytics on large content and e-commerce websites
+
+In contrast, PostHog is a product analytics platform designed to show you what your users do in your product. PostHog is event-based, so it tracks not just what pages users are visiting and how they got there, but also helps you detect, understand and fix user experience problems. 
+
+**Contents:**
+
+- [How is PostHog different to Matomo?](#how-is-posthog-different-from-matomo)
+- [PostHog vs Matomo feature comparison](#feature-comparison)
+- [Integrations and data sources](#integrations-and-data-sources)
+- [Strengths of PostHog](#strengths-of-posthog)
+- [Strengths of Matomo](#strengths-of-matomo)
+- [Coming soon to PostHog](#coming-soon-to-posthog)
+
+## How is PostHog different from Matomo?
+
+### 1. It's an all-in-one product analytics platform
+Matomo charges extra for product analytics features like funnel analysis, cohorts, path analysis, and session recording, and they're not tightly integrated. In PostHog, these are core features in all editions, including the open source release, while Experimentation is a no cost feature at [less than 1 million events per month](/pricing).
+
+### 2. Seamless integration with modern data stacks
+PostHog is built to work seamlessly with your data stack. That means we offer data import and export integrations with [most popular data warehouses](#integrations-and-data-sources), as well as numerous integrations with CRMs (Hubspot, Salesforce), data platforms (Segment, Airbyte, Rudderstack), engineering tools (PagerDuty, Sentry), and more via the [PostHog App Store](/apps).
+
+### 3. It's built for engineers
+PostHog is about giving engineering and product teams the tools they need to build better products. The core product analytics tools are part of this, but we go further by providing market-leading feature flag functionality, and integrating Session Recording so you can deploy one platform that does everything, rather than integrating multiple discrete tools into your stack.
+
+<ArrayCTA />
+
+## Feature comparison
+
+This table compares three self-hosted plans: Matomo On-Premise, PostHog Open Source, and PostHog Scale. 
+
+Matomo On-Premise ships with a robust set of core web analytics features, while advanced product analytics features are available at various prices from the Matomo On-Premise Marketplace.
+
+All features in PostHog Open Source are totally free, including core product analytics tools. 
+
+PostHog Scale is a paid edition that adds Experimentation, Group Analytics, support for multiple projects, and numerous team management tools. It's priced on a per event basis and is free up to 1 million events per month.
```

Comment:
Q: (Mainly from SEO perspective) should we link to the docs whenever we mention a feature?

Reply:
I try not to over do that, but should probably link the Experimentation product page here.